week-5/Homework/mandatory/2-exercises/exercises.js

```diff
@@ -15,6 +15,37 @@
  */
 function exerciseOne(arrayOfPeople) {
   let content = document.querySelector("#content");
+  let names = arrayOfPeople.map(p => p.name);
+  let jobs = arrayOfPeople.map(p => p.job);
+    
+  for( let i =0; i < arrayOfPeople.length; i++){
```

Comment:
Could you use something like Prettier so that you text stays nice?

https://github.com/prettier/prettier-vscode